.github/workflows/docker-aws.yml

```
@@ -0,0 +1,141 @@
+
+env:
+    node_version: 20
+    module_docker_image: "278308921038.dkr.ecr.eu-central-1.amazonaws.com/camino-suite-explorer"
+    aws_region: "eu-central-1"
+
+jobs:
+    build_docker:
+        name: Build Docker
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)